homeassistant/auth/providers/command_line.py

```diff
@@ -0,0 +1,160 @@
+"""Auth provider that validates credentials via an external command."""
+
+import typing as T
```

Comment:
I think this is not following our standard, could change to something likes
```
from typing import Any, Dict, List, Optional, cast
```

Reply:
Maybe not like it has been done for the other auth providers, but I find it awkward to mix the typing classes into global namespace. This really pollutes it, as there can be quite a lot of them laying around. It's much cleaner to hide all of this functionally unrelated stuff behind a single ``T``, imho.

Reply:
So what do you say, is changing it a strict requirement for the PR to get accepted, or do you see the advantages of not throwing every single type class into module namespace like I do?

Reply:
I cannot recall other files are using your style. Coding style could be very personal, however, we have to follow the norms in order to maintenance a readable code base.